x-pack/plugins/task_manager/server/task_scheduling.test.ts

```diff
@@ -836,16 +836,15 @@ describe('TaskScheduling', () => {
   });
 
   describe('ephemeralRunNow', () => {
-    // https://github.com/elastic/kibana/issues/181847
-    test.skip('runs a task ephemerally', async () => {
-      const ephemeralEvents$ = new Subject<TaskLifecycleEvent>();
+    test('runs a task ephemerally', async () => {
+      const ephemeralEvents$ = new BehaviorSubject<Partial<TaskLifecycleEvent>>({});
```

Comment:
The test calls `ephemeralRunNow` on an ephemeral task, which sets up a subscription to listen for ephemeral events. However, the `.subscribe` was called after the event was pushed to the observable stream, so the ephemeral task pipeline never receives it. By changing this to a `BehaviorSubject`, the last event emitted will be processed after subscribing.